source/administration/changelog.md

```diff
@@ -4,6 +4,58 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.0
+
+Release date: 2018-06-16
+
+### Highlights
+
+### Improvements
+
+ - Added plugin ability to intercept posts before they reach the DB.
+ - (Integrations) Slash commands configured to receive a GET request to get a request body.
+ - Deprecated and removed ``ExtraUpdateAt`` field from channel model.
+ - Added ``LoginIdAttribute`` to allow LDAP users to change their login ID without losing their account.
+ - Updated email notification subject line and contents for Group Messages.
+ - Added ping/pong and reconnection handling to Go WebSocket client.
+ - Added support for websocket custom dialer.
+ - Fixed ``update_status`` cluster event being sent thousands of times on restart of app servers.
+ - Added a feature to allow images uploaded to be collapsed using an icon and using ``/collapse`` command.
+ - Added a system console setting to disable the preview mode banner when email notifications are disabled.
+ - Removed license check when enforcing password requirements (moved password settings to TE).
+
+### Bug Fixes
+
+ - Fixed an issue where ``EnableUserCreation`` was set to false when not included in config.json.
+
+#### API Changes
+
+ - Improved file upload API to stream files instead of loading them entirely into memory.
+ - Removed APIv3 endpoints.
+ - Moved SAML login endpoints out of API package.
+ - Moved context.go out of api4 and into web. This includes moving params.go and some smaller http-related functions over.
+ - Created api4/handlers.go to create the API handlers using the Context and Handler from web.
+ - Created web/handlers.go to define the Handler struct, the base ServeHTTP function and a single web handler.
+ 
+### Compatibility
+ 
+#### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+
+ - Under `"?"` in `config.json`:
+   - Changed to ``"SiteURL": "",`` to
+   - Added ``"EnableAPITeamDeletion": false,`` to disable the permanent APIv4 delete team parameter.
+   - Added ``"EnablePreviewModeBanner": true,`` to
+   - Added ``"StreamingPort": 8075,`` to
+   - Added ```"MaxIdleConns": 100,`` to
+   - Added ``"MaxIdleConnsPerHost": 128,`` to
+   - Added ``"IdleConnTimeoutMilliseconds": 90000`` to
+   
+_______________   
```

Comment:
For the deprecations section, once included, add `For a list of past and upcoming deprecated features, [see our website](https://about.mattermost.com/deprecated-features/).`.

Let's include that statement in all future changelog versions too.